di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -71,17 +80,22 @@
     public Route doCallback = (Request request, Response response) -> {
         var authorizationCode = getAuthorizationCode(request);
 
-        var accessToken = exchangeCodeForToken(authorizationCode);
-
-        var userInfo = getUserInfo(accessToken);
         List<Map<String, Object>> mustacheData = new ArrayList<>();
         Map<String, Object> moustacheDataModel = new HashMap<>();
 
         try {
-            mustacheData = buildMustacheData(userInfo);
-            moustacheDataModel.put("data", mustacheData);
-        } catch (ParseException | JsonSyntaxException e) {
-            moustacheDataModel.put("error", userInfo.toJSONString());
+            var accessToken = exchangeCodeForToken(authorizationCode);
+
+            var userInfo = getUserInfo(accessToken);
+
+            try {
+                mustacheData = buildMustacheData(userInfo);
+                moustacheDataModel.put("data", mustacheData);
+            } catch (ParseException | JsonSyntaxException e) {
+                moustacheDataModel.put("error", userInfo.toJSONString());
+            }
```

Comment:
Rather than a nested try block, can we just add `catch (ParseException | JsonSyntaxException e)` to the outer try?

Reply:
Oh yeah good shout, I'll change it